README.md

```diff
@@ -1,2 +1,30 @@
 # hsp-lyrics-dataset
-Dataset for the problem of Hit Song Prediction 
+
+## Description
+
+This dataset is based on the [Hit Song Prediction (Million Song Dataset and Audio Features)](https://zenodo.org/record/3258042), which contains 95,067 songs that are representative of hits and non-hits based on their entrance or not to the Billboard Hot 100 charts. The dataset comprises release year information, high- and low-level audio features, the highest Billboard rank, if it is featured in the chart, and metadata for each song. Please refer to https://zenodo.org/record/3258042 for further information on the Hit Song Prediction dataset.
+
+We extend the aforementioned dataset with features derived from the lyrics of each sound recording. We use the [Genius](https://genius.com) database, which contains a collection of approximately 25 million songs and lyrics, from which we retrieve 11,634 song lyrics. We use multilingual BERT in order to extract the text embedding from the lyrics of each song.
```

Comment:
After the extraction of the embeddings of retrieved lyrics we deleted  the raw texts.